linux_os/guide/services/ntp/ntpd_specify_multiple_servers/rule.yml

```diff
@@ -21,7 +21,7 @@ identifiers:
     cce@rhel6: 26958-9
 
 references:
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```

Comment:
Believe there is a more specific NIST control about multiple time servers.... but having trouble finding it.